specifications/api/swagger.yaml

```
@@ -1244,7 +1244,7 @@ paths:
     get:
       operationId: getByronWalletMigrationInfo
       tags: ["Byron Wallets"]
-      summary: Migrate
+      summary: Estimate Migration Cost
```

```suggestion
      summary: Calculate Migration Cost
```

This is actually supposed to return the **exact** amount, rather than an estimation.

Hence the description below:
> Calculate the exact cost of migrating from a Byron wallet to a new wallet.

---

OK!